doc/_includes/inverse.rst

```diff
@@ -104,18 +105,51 @@ discrepancy between the measured and predicted data is tolerable.
 .. _whitening_and_scaling:
 
 Whitening and scaling
----------------------
+~~~~~~~~~~~~~~~~~~~~~
 
 The MNE software employs data whitening so that a 'whitened' inverse operator
 assumes the form
 
-.. math::    \tilde{M} = R \tilde{G}^T (\tilde{G} R \tilde{G}^T + I)^{-1}\ ,
+.. math::    \tilde{M} = M C^{^1/_2} = R \tilde{G}^\top (\tilde{G} R \tilde{G}^\top + \lambda^2 I)^{-1}\ ,
+   :label: inv_m_tilde
+
+where
+
+.. math:: \tilde{G} = C^{-^1/_2}G
+   :label: inv_g_tilde
+
+is the spatially whitened gain matrix. We arrive at the whitened inverse
+operator equation :eq:`inv_m_tilde` by making the substitution for
+`G` from :eq:`inv_g_tilde` in :eq:`inv_m` as
+
+.. math::
+
+    \tilde{M} = M C^{^1/_2} &= R G^\top (G R G^\top + \lambda^2 C)^{-1} C^{^1/_2} \\
+                             &= R \tilde{G}^\top C^{^1/_2} (C^{^1/_2} \tilde{G} R \tilde{G}^\top C^{^1/_2} + \lambda^2 C)^{-1} C^{^1/_2} \\
+                             &= R \tilde{G}^\top C^{^1/_2} (C^{^1/_2} (\tilde{G} R \tilde{G}^\top + \lambda^2 I) C^{^1/_2})^{-1} C^{^1/_2} \\
+                             &= R \tilde{G}^\top C^{^1/_2} C^{-^1/_2} (\tilde{G} R \tilde{G}^\top + \lambda^2 I)^{-1} C^{-^1/_2} C^{^1/_2} \\
+                             &= R \tilde{G}^\top (\tilde{G} R \tilde{G}^\top + \lambda^2 I)^{-1}\ .
 
-where :math:`\tilde{G} = C^{-^1/_2}G` is the spatially whitened gain matrix.
-The expected current values are :math:`\hat{j} = Mx(t)`, where :math:`x(t) =
-C^{-^1/_2}x(t)` is a the whitened measurement vector at *t*. The spatial
-whitening operator is obtained with the help of the eigenvalue decomposition
-:math:`C = U_C \Lambda_C^2 U_C^T` as :math:`C^{-^1/_2} = \Lambda_C^{-1} U_C^T`.
+The expected current values are
+
+.. math::
+   :label: inv_j_hat_t
+
+    \hat{j}(t) &= Mx(t) \\
+               &= M C^{^1/_2} C^{-^1/_2} x(t) \\
+               &= \tilde{M} \tilde{x}(t)
+
+knowing :eq:`inv_m_tilde` and taking
+
+.. math::
+   :label: inv_tilde_x_t
+
+    \tilde{x}(t) = C^{-^1/_2}x(t)
+
+as the whitened measurement vector at time *t*. The spatial
+whitening operator :math:`C^{-^1/_2}` is obtained with the help of the
+eigenvalue decomposition
+:math:`C = U_C \Lambda_C^2 U_C^\top` as :math:`C^{-^1/_2} = \Lambda_C^{-1} U_C^\top`.
```

Comment:
Perhaps, but I'd rather sort that notation change out as a separate PR. I'm trying as much as possible to stick to the notation in the existing document, and I suspect that changing this will not be trivial.